<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Caudal In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Caudal sal(L)</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Tank">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditcap">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad(L)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditout">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderout">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>265</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TimeOut</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdittime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditcap</tabstop>
  <tabstop>lineEditin</tabstop>
  <tabstop>lineEditout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
